shell/platform/fuchsia/flutter/gfx_external_view_embedder.cc

```diff
@@ -491,7 +526,7 @@ void GfxExternalViewEmbedder::SubmitFrame(
             embedded_views_height;
         auto& scenic_layer = scenic_layers_[scenic_layer_index];
         auto& scenic_rect = found_rects->second[rect_index];
-        scenic_layer.shape_node.SetLabel("Flutter::Layer");
+        scenic_layer.shape_node.SetLabel("Flutter::Layer::Image");
```

Comment:
Can you please nest each Flutter::Layer::Image and its associated Flutter::Layer::HitRegions underneath a "Flutter::Layer" EntityNode, for debuggability?